<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="41260" started-at="2020-07-15T19:03:36Z" finished-at="2020-07-15T19:04:17Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="41260" started-at="2020-07-15T19:03:36Z" finished-at="2020-07-15T19:04:17Z">
      <class name="com.crm.qa.test.LoginTest">
        <test-method status="PASS" signature="setExtent()[pri:0, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="setExtent" is-config="true" duration-ms="177" started-at="2020-07-16T00:33:36Z" finished-at="2020-07-16T00:33:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtent -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="setup" is-config="true" duration-ms="12777" started-at="2020-07-16T00:33:36Z" finished-at="2020-07-16T00:33:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="validatePageTitle()[pri:1, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="validatePageTitle" duration-ms="28" started-at="2020-07-16T00:33:49Z" finished-at="2020-07-16T00:33:49Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Cogmento CRM1] but found [Cogmento CRM]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Cogmento CRM1] but found [Cogmento CRM]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.crm.qa.test.LoginTest.validatePageTitle(LoginTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validatePageTitle -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="tearDown" is-config="true" duration-ms="1092" started-at="2020-07-16T00:33:49Z" finished-at="2020-07-16T00:33:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validatePageTitle status=FAILURE method=LoginTest.validatePageTitle()[pri:1, instance:com.crm.qa.test.LoginTest@6f7fd0e6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="setup" is-config="true" duration-ms="11296" started-at="2020-07-16T00:33:50Z" finished-at="2020-07-16T00:34:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="validateInvalidLogin()[pri:2, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="validateInvalidLogin" duration-ms="1322" started-at="2020-07-16T00:34:01Z" finished-at="2020-07-16T00:34:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateInvalidLogin -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="tearDown" is-config="true" duration-ms="1302" started-at="2020-07-16T00:34:03Z" finished-at="2020-07-16T00:34:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validateInvalidLogin status=SUCCESS method=LoginTest.validateInvalidLogin()[pri:2, instance:com.crm.qa.test.LoginTest@6f7fd0e6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="setup" is-config="true" duration-ms="11344" started-at="2020-07-16T00:34:04Z" finished-at="2020-07-16T00:34:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="validateLogin()[pri:2, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="validateLogin" duration-ms="1398" started-at="2020-07-16T00:34:15Z" finished-at="2020-07-16T00:34:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLogin -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="tearDown" is-config="true" duration-ms="675" started-at="2020-07-16T00:34:17Z" finished-at="2020-07-16T00:34:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validateLogin status=SUCCESS method=LoginTest.validateLogin()[pri:2, instance:com.crm.qa.test.LoginTest@6f7fd0e6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endreport()[pri:0, instance:com.crm.qa.test.LoginTest@6f7fd0e6]" name="endreport" is-config="true" duration-ms="370" started-at="2020-07-16T00:34:17Z" finished-at="2020-07-16T00:34:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endreport -->
      </class> <!-- com.crm.qa.test.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
